PHYSICS/AddRope.md

```diff
@@ -35,4 +85,46 @@ Rope will sometimes contract and fall to the ground like you'd expect it to, but
 * **unkPtr**: Unknown pointer, always 0 in original scrips.
 
 ## Return value
-A script handle for the rope
\ No newline at end of file
+A script handle for the rope
+
+## Examples
+```lua
+RegisterCommand("new_rope", function(source, args, rawCommand)

```

Comment:
Examples shouldn't be in a register command, please remove it and remove the extra indentation.

Reply:
Out of curiosity, is there any reason why examples shouldn't be in commands? Just seems to me that an example command of how to use the native, in this case showing how to spawn a simple rope, would be a useful way to show an example of the native, so is there any particular reason or angle I may not be seeing as to why commands are not ideal examples?

Reply:
Sorry for the late reply here, theres a few reasons here

1. The code is straight forward and only includes what it needs
2. The code becomes easier to copy & paste for people who are just starting and want to figure out what certain things do
3. We only have so much code space in the native docs currently, so the less space used by indention, the better.

Reply:
Good to know, thanks for the insight!